### PR TITLE
[MM-48578] - Code formatting for pascal and pas

### PR DESCRIPTION
**Pull request by hchorfispiria**
#### Summary
Code formatting has support for delphi, but not pascal and pas for pascal.

QA test steps:
- Create a new message by specifiying pas and pascal in a ``` code block.
- Similalery to delphi, as a result the code pascal or pas should be highlighted.

#### Ticket Link

  Fixes https://github.com/mattermost/mattermost-server/issues/21711

#### Screenshots
pascal
pas
Result:
result

#### Release Note

```release-note
Highlight pas and pascal code block.
```

### CODE REVIEW

File: webapp/channels/src/utils/constants.tsx [hunk 1]
```diff
@@ -1822,7 +1822,7 @@ export const Constants = {
         css: {name: 'CSS', extensions: ['css']},
         d: {name: 'D', extensions: ['d', 'di'], aliases: ['dlang']},
         dart: {name: 'Dart', extensions: ['dart']},
-        delphi: {name: 'Delphi', extensions: ['delphi', 'dpr', 'dfm', 'pas', 'pascal', 'freepascal', 'lazarus', 'lpr', 'lfm']},
+        delphi: {name: 'Delphi', extensions: ['delphi', 'dpr', 'dfm', 'pas', 'pascal', 'freepascal', 'lazarus', 'lpr', 'lfm'], aliases: ['pas', 'pascal']},
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/utils/constants.tsx:1825]: I understand that Pascal has very few differences from Delphi. By this, we are making sure both are the same.

Comment (M-ZubairAhmed) [reply webapp/channels/src/utils/constants.tsx:1825]: never mind, i guess this is ok

**Review by M-ZubairAhmed — APPROVED** [review]

**Comment by M-ZubairAhmed** [comment]
@hchorfispiria you need to sign the cla

**Comment by hchorfispiria** [comment]
@M-ZubairAhmed I signed the CLA

**Review by hmhealey — APPROVED** [review]
Thanks for the PR, @hchorfispiria. This looks good to me!

I would've actually thought that highlight.js had separate support for Pascal, but it doesn't appear to

**Comment by hmhealey** [comment]
I'm going to update this to the latest master since there's something on the server-side causing the linting to fail...

/update-branch

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**